.changes/unreleased/tinymce-TINY-10694-2024-02-28.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Changed
+body: Updated deprecation/removed console message for 7.0
```

All of the changes are for 7 not sure we need to state that.

```suggestion
body: Updated deprecation/removed console message.
```